electrumx/server/block_processor.py

```diff
@@ -513,17 +514,18 @@ def backup_txs(self, txs, is_unspendable):
 
                 # Get the hashX
                 hashX = script_hashX(txout.pk_script)
+
                 cache_value = spend_utxo(tx_hash, idx)
-                touched.add(cache_value[:-13])
+                touched.add(cache_value[:-15])
```

Comment:
15 appears in multiple places. Maybe it's worth making it a named constant

Reply:
best part is, these are 3 constants - sum together